bosh-setup/manifests/multiple-vm-cf.yml

```diff
@@ -320,6 +456,153 @@ properties:
     port: 5524
     roles:
     - {name: ccadmin, password: c1oudc0w, tag: admin}
+  consul:
+    agent:
+      log_level: null
+      servers:
+        lan:
+        - REPLACE_WITH_CONSUL_IP
+    agent_cert: |+
+      -----BEGIN CERTIFICATE-----
+      MIIEITCCAgugAwIBAgIQTDMCJHBiQrY8hlM78doHTzALBgkqhkiG9w0BAQswEzER
```

Comment:
They are copied from community manifests and only can be accessed by internal trusted components. And these examples are for testing but not for production.

Best Regards,
Abel HU

? 2016?3?10??16:56?Kay Singh <notifications@github.com<mailto:notifications@github.com>> ???


In bosh-setup/manifests/multiple-vm-cf.yml<https://github.com/Azure/azure-quickstart-templates/pull/1371#discussion_r55649462>:

> @@ -320,6 +456,153 @@ properties:
>      port: 5524
>      roles:
>      - {name: ccadmin, password: c1oudc0w, tag: admin}
> +  consul:
> +    agent:
> +      log_level: null
> +      servers:
> +        lan:
> +        - REPLACE_WITH_CONSUL_IP
> +    agent_cert: |+
> +      -----BEGIN CERTIFICATE-----
> +      MIIEITCCAgugAwIBAgIQTDMCJHBiQrY8hlM78doHTzALBgkqhkiG9w0BAQswEzER


Is it ok to have private keys/certs like this?

-
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/1371/files#r55649462>.